root_orchestrator/cloud-scheduler/resource_management/resource_helper.py

```diff
@@ -0,0 +1,12 @@
+from _datetime import datetime
+
+CLUSTERS_FRESHNESS_INTERVAL = 30
+
+def is_cluster_active(cluster):
+    print('check cluster activity...')
```

Comment:
Because we are already using loggers in many places I think it would also make sense to use them here:
Nice resources: 
- https://docs.python.org/3/howto/logging.html
- https://docs.python.org/3/library/logging.html

@giobart do we have any "inofficial" guidelines regarding logging? E.g. when/is it ok to use plain prints?

Reply:
This was copied from some other file, so didn't really look at the implementation of the file. However, I agree with you that we should have a more elagant way of logging. Happy to use the logger if we have a particular way of logging in the project.